modules/hitlnext/src/backend/api.ts

```diff
@@ -357,4 +357,13 @@ export default async (bp: typeof sdk, state: StateType) => {
       res.send(messages)
     })
   )
+
+  router.delete(
```

Comment:
No one told you that I know but we don't use delete routes because of some paid clients security policies. In some old webserver implementations, DELETE and PUT methods were not protected and allowed to delete resources (static files) in no time.

so please use /delete verb at the end of your path. in this case /handoffs/:id/delete

Reply:
Ah perfect! Thanks for the explanation, I'll keep that in mind for next time!